README.Rmd

```diff
@@ -131,8 +207,16 @@ Visually, this looks like:
 
 ![](docs/figures/valuation_perc_owner.png)
 
+For what the office terms "nonlivable" spaces, i.e. parking spaces, storage space, and common area, the breakout of value works differently. See [this excel sheet](docs/spreadsheets/condo_nonlivable_demo.xlsx) for an interactive example of how nonlivable spaces are valued based on the total value of a building's livable space.
```

Comment:
**nitpick:** Mega grammar nitpick:

```suggestion
For what the office terms "nonlivable" spaces, e.g. parking spaces, storage space, and common area, the breakout of value works differently. See [this excel sheet](docs/spreadsheets/condo_nonlivable_demo.xlsx) for an interactive example of how nonlivable spaces are valued based on the total value of a building's livable space.
```

Reply:
You know, I'm not entirely sure I agree with this since the list is exhaustive, not just some examples. But I'll change it to something else even cooler instead.